### PR TITLE
CommonServer - add underscore to camel case converter

### PR DESCRIPTION
**Pull request by shaniacht1**

### CODE REVIEW

File: Scripts/script-CommonServer.yml [hunk 1]
```diff
@@ -203,6 +203,18 @@ script: |-
...
   }
 
+  var tblToMd = tableToMarkdown;
+
+  /**
+   * Converts underscore case strings to camel case
+   * @param {String} string - string to be converted - i.e. hello_world
+   * @returns {String} - converted string - i.e. HelloWorld
+  */
+  var undrscrToCamelCase = function(string){
```
Comment (dorsha) [inline Scripts/script-CommonServer.yml]: There is a typo in the function name, should be `underscoreToCamelCase`

File: Scripts/script-CommonServer.yml [hunk 2]
```diff
@@ -203,6 +203,18 @@ script: |-
...
+      string = '_'+string;
```
Comment (dorsha) [inline Scripts/script-CommonServer.yml]: In JavaScript (and other languages), it is not a good practice to mutate the function input, so please change the assignment to another variable.

File: Scripts/script-CommonServer.yml (hunk 1 above)
Comment (shaniacht1) [inline Scripts/script-CommonServer.yml]: done

File: Scripts/script-CommonServer.yml (hunk 2 above)
Comment (shaniacht1) [inline Scripts/script-CommonServer.yml]: done

**Pull request merged**